uktrade/data-hub-frontend
# Firebreak Q3 2024 fix a11y issues

## Description of change

PR fixes a number of accessibility issues as part of the Q3 2024 firebreak project.

## Test instructions

Spin up the sandbox (or other instance) and visit the pages listed in the screenshots section. The pages have been grouped by similar issues and I've only included one screenshot for each group.

Only the investment projects pages have had a small visual change to the contrast of headings. All other pages should be identical.

## Screenshots
<details>

/
Fixed page heading level and changed landmark region.
![image](https://github.com/user-attachments/assets/cfcfb5fa-8e86-4ccd-bb25-b7b40c23cbd1)

/investment-projects/
Fixed heading level, landmark region and svg img alt (changes on filter changes).
![image](https://github.com/user-attachments/assets/b4e71e97-8ff1-4074-812d-6adde95e9ad7)

/my-tasks/
/my-referrals/
Fixed page heading level and changed landmark region
![image](https://github.com/user-attachments/assets/945b1ab7-ae5b-4f3b-a234-ecd796905dfb)

/community/
Fixed page heading level
![image](https://github.com/user-attachments/assets/1dbc1e12-4e0f-4d1b-8676-d6a850122609)

/investments/projects/133a3815-8850-4568-b491-3ad6b011057f/details
/investments/projects/133a3815-8850-4568-b491-3ad6b011057f/edit-details
/investments/projects/133a3815-8850-4568-b491-3ad6b011057f/edit-requirements
/investments/projects/133a3815-8850-4568-b491-3ad6b011057f/edit-team-members
/investments/projects/133a3815-8850-4568-b491-3ad6b011057f/edit-value
/investments/projects/133a3815-8850-4568-b491-3ad6b011057f/team
/investments/projects/133a3815-8850-4568-b491-3ad6b011057f/edit-client-relationship-management
/investments/projects/create/dfd02cf1-3192-4057-b966-c217f81a64b3
/investments/projects/133a3815-8850-4568-b491-3ad6b011057f/edit-project-management
/investments/projects/133a3815-8850-4568-b491-3ad6b011057f/remove-ukcompany
/investments/projects/133a3815-8850-4568-b491-3ad6b011057f/evaluation
/investments/projects/133a3815-8850-4568-b491-3ad6b011057f/edit-history
Fixed colour contrast
![image](https://github.com/user-attachments/assets/4c52464c-d33d-4db9-b573-879f69375e46)

/investments/projects/133a3815-8850-4568-b491-3ad6b011057f/find-ukcompany
/investments/projects/133a3815-8850-4568-b491-3ad6b011057f/find-associated
Fixed empty heading
![image](https://github.com/user-attachments/assets/99008dc1-7ead-4181-9689-5fbd60868853)

/omis/create
Fixed empty heading
![image](https://github.com/user-attachments/assets/72011f89-5b77-4740-b936-40fe06b0d34b)

/export/
Fixed heading levels and landmark regions
![image](https://github.com/user-attachments/assets/334cfd2a-9fe4-4aa8-bd5f-67956586748f)

/reminders/settings/my-tasks-task-completed/
Fixed heading levels and landmark regions
![image](https://github.com/user-attachments/assets/b34bc6ec-e051-42d0-9700-8622038b1f98)

/reminders/settings/my-tasks-task/overdue/
/reminders/settings/my-tasks-task-amended-by-others/
/reminders/settings/my-tasks-task-assigned-to-me-from-others/
/reminders/settings/my-tasks-task-due-date-approaching/
Fixed heading level order
![image](https://github.com/user-attachments/assets/782076d3-1af8-4275-b2a1-2f0db9469b7f)

/reminders/settings/companies-new-interactions/
/reminders/settings/companies-no-recent-interactions/
/reminders/settings/investments-no-recent-interactions/
Fixed heading level order and field label
![image](https://github.com/user-attachments/assets/d1140e33-b076-4408-8be9-5b64ab7d2ed9)

/reminders/settings/
/reminders/settings/investments-estimated-land-dates/
Fixed heading level order
![image](https://github.com/user-attachments/assets/1e8385db-38df-4cbc-97e7-d93d27aec9d5)

/reminders/
/reminders/investments-estimated-land-dates/
/reminders/investments-no-recent-interactions/
/reminders/investments-outstanding-propositions/
/reminders/companies-no-recent-interactions/
/reminders/companies-new-interactions/
/reminders/my-tasks-due-date-approaching/
/reminders/my-tasks-task-assigned-to-me-from-others/
/reminders/my-tasks-task-amended-by-others/
/reminders/my-tasks-task-overdue/
/reminders/my-tasks-task-completed/
Fix list items not within ul element.
![image](https://github.com/user-attachments/assets/019a5777-92bb-43e9-a0de-68724379a1b2)

/omis/:orderId/reconciliation/payment-receipt
Fixed empty header on table
![image](https://github.com/user-attachments/assets/0007ae28-ecce-4462-bba9-b70c9681535f)

/omis/7ea3943c-6220-41c8-b704-ff0c4b33b346/edit/complete-order
/omis/7ea3943c-6220-41c8-b704-ff0c4b33b346/edit/assignee-time
Fixed empty header on table and input field labels.
![image](https://github.com/user-attachments/assets/7bff926d-86ad-4204-99ba-cf15ae372696)

/omis/7ea3943c-6220-41c8-b704-ff0c4b33b346/payment-receipt
Fixed empty header on table
![image](https://github.com/user-attachments/assets/53164543-bd8a-408a-a2cc-d35563ab479e)

/omis/7ea3943c-6220-41c8-b704-ff0c4b33b346
/omis/7ea3943c-6220-41c8-b704-ff0c4b33b346/work-order
Fix list items not within ul element.
![image](https://github.com/user-attachments/assets/ddd09d14-88f0-43c0-b35c-0cd4628dac27)

</details>
 
## Checklist

[//]: # 'When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/main/docs/Code%20review%20guidelines.md'
